audition/distance_from_best.py

```diff
@@ -33,11 +33,23 @@ def _create(self):
             train_end_time timestamp,
             metric text,
             parameter text,
-            raw_value float,
-            below_best float,
-            below_best_next_time float
+            dist_from_abs_worst float,
```

Comment:
In other words:

Are you proposing we don't create `dist_from_abs_worst`? If so I have no idea how we'd do any of the weighted selection rules without defining the range upfront.

Or are you proposing we create that column, and generally use it, and just keep `raw_value` around for rules like variance that aren't directional?